services/121-service/seed-data/program/program-pilot-nl-2.json

```diff
@@ -105,13 +105,7 @@
         "en": "Partner Organization"
       },
       "name": "namePartnerOrganization",
-      "type": "string",
-      "export": [
-        "all-people-affected",
-        "included",
-        "selected-for-validation",
-        "payment"
-      ]
+      "type": "string"
```

Comment:
This would become "`text`" instead of `string`, right?

See: https://github.com/global-121/121-platform/pull/2990/files#diff-04e9fd4a0e11700f1f08ed3c90e669d605859ef569543a9e95c93ecdc15dd59eR5